graphql_compiler/schema_generation/schema_graph.py

```diff
@@ -537,13 +580,13 @@ def _set_up_schema_elements_of_kind(self, class_name_to_definition, kind, class_
                         property_name_to_descriptor[property_name] = property_descriptor
 
                 if (property_name not in property_name_to_descriptor and not abstract and
-                        kind == SchemaElement.ELEMENT_KIND_EDGE):
+                        kind == Edge):
```

Comment:
It's almost never a good idea to compare types for equality. Use `issubclass` instead: https://docs.python.org/3/library/functions.html#issubclass